release-notes.md

```diff
@@ -9,7 +9,27 @@ meta:
 ---
 # Release Notes
 
+## 2.3.0
+### Admission Controller
```

Comment:
I'm trying to be more upfront about the problem we're solving with new features. How does this sound?
----
As companies roll out Kubernetes across multiple clusters and teams, configuration consistency becomes important. Admission Controllers in Kubernetes can apply out-of-the-box guardrails (known as "policies") to new deployments that ensure security, compliance, and workload efficiency -- without sacrificing the need for velocity and time to market. Of course, the Fairwinds Insights Admission Controller also supports custom policies through the Open Policy Agent (OPA) report.

We've rolled out a new UI called "Admission Controller" to enable users to view deployment activity, as well as which deployments the Admission Controller is allowing vs. denying. If a resource is denied from your cluster, you will now see all of the action items associated with that event, along with a remediation recommendation.

Reply:
I think this might be good for the email, but I'd like to keep the release notes technical and matter-of-fact.